velox/exec/TableWriter.h

```diff
@@ -104,6 +104,9 @@ class TableWriter : public Operator {
  private:
   void createDataSink();
 
+  // Updates the OperatorStats physicalWrittenBytes with current written bytes
```

Comment:
```
// Updates physicalWrittenBytes in the operator stats after adding each input.
```

Reply:
I will leave it as it is because this function can be called at multiple places, not only for adding input

Reply:
How about:
```
// Updates physicalWrittenBytes in OperatorStats with current written bytes.
```
Thanks!